site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,119 @@
+- Call [`navigator.storage.persist()`][storage-persist] for protection against eviction.
+
+Extension storage is shared across the extension's origin including the extension service worker,
+any extension pages (including popups and the side panel), and offscreen documents. In content
+scripts, calling web storage APIs accesses data from the host page the content script is injected on
+and not the extension.
+
+### Access in service workers {: #storage-in-service-workers }
+
+The [IndexedDB][indexeddb] and [Cache Storage][cache-storage] APIs are accessible in service
+workers. However, [Local Storage][local-storage] and [Session Storage][session-storage] are not.
+
+If you have a specific use case, use an [offscreen document][offscreen] to access these.
+
+### Partitioning {: #storage-partitioning }
+
+Partitioning is where keys are introduced for stored data to limit where it can be accessed. Storage
+has historically been keyed by origin.
+
+Starting in Chrome 115, [storage partitioning][storage-partitioning] introduces changes to how
+partitioning keys are defined to prevent certain types of cross-site tracking. In practice, this
+means that if site A embeds an iframe containing site B, site B will not be able to access the same
+storage it would usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two exemptions apply:
+
+- If a page with the `chrome-extension://` scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its top-level partition.
+- If a page with the `chrome-extension://` scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its top-level partition.
+
+## Cookies {: #cookies }
+
+Cookies provide a way to store key/value pairs associated with a specific domain and path. They have
+limited value in extensions but understanding their behavior can be useful if you have a specific
+use case or have bundled a third-party script which uses them in its implementation.
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the `https://`
+scheme. Consequently, `chrome-extension://` pages are not able to set cookies with this attribute.
+
+This also means that extension pages cannot use other cookie attributes where the `Secure` attribute is
+required:
+
+- [`SameSite=None`][same-site]
+- [`Partitioned`][chips]
+
+### Partitioning {: #cookies-partitioning }
+
+{% Aside %}
+See [https://crbug.com/1463991](https://crbug.com/1463991) which tracks possible changes to the
+behaviour when an extension with host permissions embeds a third-party site.
+{% endAside %}
+
+As mentioned [above](#secure-cookies), extensions cannot create partitioned cookies, and cookies
+are not currently partitioned by default (see the Privacy Sandbox
```

Comment:
```suggestion
are not partitioned by default (see the Privacy Sandbox
```

Reply:
Made significant changes to this section.